.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.idea/
```

Comment:
I guess this is not necessary.

Reply:
It should be added do developer's global .gitignore. In repository .gitignore should be only files that are project oriented only